sonic-slave-buster/Dockerfile.j2

```diff
@@ -375,7 +375,7 @@ RUN sudo augtool --autosave "set /files/etc/dpkg/dpkg.cfg/force-confold"
 RUN apt-get -y build-dep linux
 
 # For gobgp and telemetry build
-RUN export VERSION=1.14.2 \
+RUN export VERSION=1.15.15 \
```

Comment:
hi @qiluo-msft , since cherypick is failed on https://github.com/sonic-net/sonic-buildimage/pull/14637, I reopen this PR instead, could you pls help on review? thanks.